packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -0,0 +1,140 @@
+  NEUTRAL: 'neutral',
+  EMPHASIS: 'emphasis',
+  GHOST: 'ghost',
+};
+
+const propTypes = {
+  /**
+   * The options to display in the dropdown. Should be comprised of the subcomponent `Button`.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Determines whether the dropdown should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Determines whether the component should have block styles applied. The dropdown will match the component's width.
+   */
+  isBlock: PropTypes.bool,
+  /**
+   * Sets the text that will be shown on the dropdown button.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Sets the styles of the component. The default is 'neutral'.
+   * Must be one of 'neutral', 'emphasis', or 'ghost'.
+   */
+  variant: PropTypes.oneOf(Object.values(Variants)),
+};
+
+const defaultProps = {
+  disabled: false,
+  isBlock: false,
+  variant: 'neutral',
+};
+
+class DropdownButton extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleDropdownButtonClick = this.handleDropdownButtonClick.bind(this);
+    this.handleDropdownRequestClose = this.handleDropdownRequestClose.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleKeyUp = this.handleKeyUp.bind(this);
+
+    this.state = { isOpen: false, active: false };
+  }
+
+  handleDropdownButtonClick() {
+    this.setState(prevState => ({ isOpen: !prevState.isOpen }));
+  }
+
+  handleDropdownRequestClose() {
+    this.setState({ isOpen: false });
+  }
+
+  /*
+    In FireFox active styles don't get applied on space
+   */
+  handleKeyDown(event) {
+    if (event.keyCode === KeyCode.KEY_SPACE) {
+      this.setState({ active: true });
+    }
+  }
+
+  handleKeyUp(event) {
+    if (event.keyCode === KeyCode.KEY_SPACE) {
+      this.setState({ active: false });
+    }
+  }
+
+  render() {
+    const {
+      children,
+      disabled,
+      isBlock,
+      label,
+      variant,
+      ...customProps
+    } = this.props;
+
+    const {
+      isOpen,
+      active,
+    } = this.state;
+
+    const classnames = cx(
+      'dropdown-button',
+      variant,
+      { 'is-active': isOpen || active },
+      { 'is-block': isBlock },
+    );
+
+    return (
+      <DropdownButtonBase
+        {...customProps}
+        buttons={children}
+        isOpen={isOpen}
+        requestClose={this.handleDropdownRequestClose}
+        disabled={disabled}
+        isBlock={isBlock}
+      >
+        <button
+          type="button"
+          className={classnames}
+          onClick={this.handleDropdownButtonClick}
+          onKeyDown={this.handleKeyDown}
+          onKeyUp={this.handleKeyUp}
+          disabled={disabled}
+          tabIndex={disabled ? '-1' : undefined}
+          aria-disabled={disabled}
+          aria-expanded={isOpen || undefined}
+          aria-haspopup
+        >
+          <span className={cx('dropdown-button-text')}>{label}</span>
+          <span className={cx('chevron-icon')} />
```

Comment:
Looking at the call outs, it looks like this should be using the caret down icon, not the chevron

Reply:
Fixed in aca4376fcf55905535d77b2168118fd85dfb4a15